cli/utils.go

```diff
@@ -168,7 +169,7 @@ func setContextIdentity(cmd *cobra.Command, privateKeyHex string) error {
 		return err
 	}
 
-	ctx := db.SetContextIdentity(cmd.Context(), immutable.Some(identity))
+	ctx := identity.WithContext(cmd.Context(), immutable.Some(ident))
```

Comment:
question: Sorry reviewed a bit fast, lost context, what happened to `db.SetContext...`?

Reply:
I don't see a reason to depend on `db` just to get an identity from a `Context`

Reply:
Ah I see! thanks for clarification.